Index: Semana.7POO.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Semana.7POO.py b/Semana.7POO.py
--- a/Semana.7POO.py	
+++ b/Semana.7POO.py	(date 1721493415842)
@@ -5,9 +5,12 @@
         self.nombre = nombre
         self.fundacion = fundacion
         print(f"Marca '{self.nombre}' fundada en {self.fundacion}.")
-# El destructor (__del__) de Marca está configurado para imprimir un mensaje cuando se elimina la instancia de la marca.
+
+    # El destructor (__del__) de Marca está configurado para imprimir un mensaje cuando se elimina la instancia de la marca.
     def __del__(self):
         print(f"Marca '{self.nombre}' eliminada.")
+
+
 # Esta clase tiene un constructor (__init__) que inicializa el modelo de la motocicleta, el año y la instancia
 # de Marca a la que pertenece.
 # Cuando se crea una instancia de Motocicleta, se imprime un mensaje indicando la creación de la motocicleta y
@@ -18,11 +21,13 @@
         self.anio = anio
         self.marca = marca
         print(f"Motocicleta {self.modelo} del año {self.anio} creada, de la marca '{self.marca.nombre}'.")
-# El destructor (__del__) de Motocicleta está configurado para imprimir un mensaje cuando
-# se elimina la instancia de la motocicleta.
+
+    # El destructor (__del__) de Motocicleta está configurado para imprimir un mensaje cuando
+    # se elimina la instancia de la motocicleta.
     def __del__(self):
         print(f"Motocicleta {self.modelo} del año {self.anio} eliminada.")
 
+
 # Creación de instancias de Marca y Motocicleta
 
 mi_marca = Marca("Honda", 1948)
